.github/workflows/publish_remote_config.yml

```
@@ -0,0 +1,30 @@
+name: Publish remote config
+
+on:
+  push:
+    branches:
+      - master
```

Maybe add a path here so it only runs if `reference-data` is updated?

Haven't tested it out, but something like:

```suggestion
  push:
    branches:
      - master
    paths:
      - 'src/reference-data/*.json'
```

---

Looks like the workflow docs use double star for wildcards. Will do some local testing.